antarest/study/business/areas/renewable_management.py

```diff
@@ -114,7 +114,7 @@ def create_renewable_output(
     cluster_id: str,
     config: t.Mapping[str, t.Any],
 ) -> "RenewableClusterOutput":
-    obj = create_renewable_config(study_version=StudyVersion.parse(study_version), **config, id=cluster_id)
+    obj = create_renewable_config(study_version=StudyVersion.parse(study_version), **config, id=cluster_id.lower())
```

Comment:
Should be enforced in the renawable config class instead (actually in ItemProperties)